test/test_ops.py

```diff
@@ -2195,7 +2195,22 @@ def test_fake_crossref_backward_no_amp(self, device, dtype, op):
     @skipOps('TestFakeTensor', 'test_fake_crossref_backward_amp', fake_backward_xfails | fake_autocast_backward_xfails)
     def test_fake_crossref_backward_amp(self, device, dtype, op):
         self._test_fake_crossref_helper(device, dtype, op, torch.cuda.amp.autocast)
-
+    def test_empty_strided(self, device):
+        result = torch.empty((3, 3), layout=torch.strided, device=device)
```

Comment:
@alhridoy  Because the default layout for the`empty` and other factory functions is `torch.strided`, is it necessary to specify `layout=torch.strided` explicitly here? Without the layout argument to `empty`, the layout of the result should be `torch.strided`.  I think that needs to be tested here. Let me know if I am missing anything here.

Reply:
Hi @akdong91 , Thanks for the feedback. I've updated the tests based on your feedback. I've added a generic test that checks all factory functions with layout=torch.strided specified. This test also checks that the default layout is torch.strided when the layout argument is not provided. Please take a look and let me know if there are any further changes needed.